integration-tests/cucumber-tests-platform-smartmetering/src/test/resources/features/osgp-adapter-ws-smartmetering/bundle/BundledDeCoupleMBusDeviceByChannel.feature

```diff
@@ -0,0 +1,135 @@
+@SmartMetering @Platform
+Feature: SmartMetering Bundle - De Couple M-Bus Device By Channel
+  As a grid operator
+  I want to be able to de couple an M-Bus device by channel to a smart meter
+
+  Scenario: DeCouple Mbus Device By Channel on a administratively decoupled E-meter
+    Given a bundle request
+      | DeviceIdentification | TEST1024000000001 |
+    And the bundle request contains a decouple mbus device by channel action
```

Comment:
de couple and decouple used intermittently.

Reply:
All using _Decouple M-Bus Device By Channel_ now